Assets/Scripts/UnityServices/Lobbies/LobbyServiceFacade.cs

```diff
@@ -212,8 +212,14 @@ public async Task<(bool Success, Lobby Lobby)> TryQuickJoinLobbyAsync()
         void ResetLobby()
         {
             CurrentUnityLobby = null;
-            m_LocalUser.ResetState();
-            m_LocalLobby?.Reset(m_LocalUser);
+            if (m_LocalUser != null)
+            {
+                m_LocalUser.ResetState();
+            }

```

Comment:
There was no null-conditional operator for m_LocalUser, was this change made on purpose?

Reply:
Yes I added this for consistency. Although now that I think back, this should never be null. It might be better to let it throw an exception if it is. Same with m_LocalLobby

Reply:
I think this could apply to some of the other null checks as well

Reply:
throwing an exception sounds good